util/proposal.go

```diff
@@ -34,5 +40,23 @@ func GetProposalName(payload string) (string, error) {
 
 // IsValidProposalName reports whether str is a valid proposal name
 func IsValidProposalName(str string) bool {
+	regexMounted := MountValidRegExpProposalName()
+	validProposalName := regexp.MustCompile(regexMounted)
```

Comment:
The regexp should probably be compiled in the [`init` function](https://golang.org/doc/effective_go.html#init) so it is only executed once.